### PR TITLE
You have tried to upload file(s) that are not safe.

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #8903

### Summary of Changes

If there is no info returned from ` JInputFiles get()` you have tryed to upload unsave content. -> Tell that to the user.

### Testing Instructions

Try to upload a zip file containing php scripts e.g. a extension using com_media.

### Expected result

Error message: `You have tryed to upload a save file that is not save.`

### Actual result

No message

### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_media.ini [hunk 1]
```diff
@@ -40,6 +40,7 @@ COM_MEDIA_ERROR_UNABLE_TO_DELETE=" Unable to delete:&#160;"
 COM_MEDIA_ERROR_UNABLE_TO_UPLOAD_FILE="Unable to upload file."
 COM_MEDIA_ERROR_UPLOAD_INPUT="Please input a file to upload"
 COM_MEDIA_ERROR_WARNFILENAME="File name must only contain alphanumeric characters and no spaces."
+COM_MEDIA_ERROR_WARNFILENOTSAFE="You have tryed to upload file(s) that are not safe."
```
Comment (julienV) [inline administrator/language/en-GB/en-GB.com_media.ini]: should be 'you have tried...'

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_media.ini]: correct it should be tried

Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_media.ini]: Thanks fixed with the last commit.

**Comment by julienV** [comment]
this only fixes the issue for upload done in com_media, but not from other code using jinputfile get function... I still think raising an exception there is necessary, and here you should catch it instead of testing for empty $files

**Comment by zero-24** [comment]
i have just replyed to the other issue please open a PR with that exeption against the 4.0 branch and test the temp fix here. Thanks

**Comment by franz-wohlkoenig** [comment]
@zero-24 can you say a extension using com_media to test correctly?

**Comment by zero-24** [comment]
You can try to upload any extension (zip file) using com_media. I have tried a custom extension one.

**Comment by franz-wohlkoenig** [comment]
@zero-24 can you give me a Name for an Extension using com_media?

**Comment by zero-24** [comment]
Ah than you missunderstood that. You should just try to upload a zip file containing a extension. (or any other PHP code)

Like the weblinks package. As this is marked as unsafe file. The extension itself do not matter :)

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 1bc78ff4ba5a0975e5ddca8905dbda7bb3e3cf86<br /><br />Tried upload com_patchtester.zip in "Media", got expected Error Message.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15466">issues.joomla.org/tracker/joomla-cms/15466</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
Please reassign Milestone 3.7.1. Was removed cause set "Easy Test" on "Yes".

File: administrator/language/en-GB/en-GB.com_media.ini [hunk 2]
```diff
@@ -40,7 +40,7 @@ COM_MEDIA_ERROR_UNABLE_TO_DELETE=" Unable to delete:&#160;"
 COM_MEDIA_ERROR_UNABLE_TO_UPLOAD_FILE="Unable to upload file."
 COM_MEDIA_ERROR_UPLOAD_INPUT="Please input a file to upload"
 COM_MEDIA_ERROR_WARNFILENAME="File name must only contain alphanumeric characters and no spaces."
-COM_MEDIA_ERROR_WARNFILENOTSAFE="You have tryed to upload file(s) that are not safe."
+COM_MEDIA_ERROR_WARNFILENOTSAFE="You have tried to upload file(s) that are not safe."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_media.ini]: How about `This file(s) is unsafe to upload.` to be consistent with sentence structure as the following line?

**Comment by zero-24** [comment]
Would be great to get some testers here? ;)

**Comment by wojsmol** [comment]
@zero-24 I will test this, give me one hour :)

**Comment by wojsmol** [comment]
I have tested this item :white_check_mark: successfully on 9304662a88f79be462bce5ee7e7cf4d33b5b3384<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15466">issues.joomla.org/tracker/joomla-cms/15466</a>.</sub>

**Comment by zero-24** [comment]
Thanks.

**Pull request merged**